CHANGELOG.md

```diff
@@ -18,7 +16,7 @@
   - `sourced status config` shows the contents of the Docker Compose environment variables. This is useful, for example, to check if the active working directory was configured to include or skip forks when downloading the data from GitHub
   - `sourced status all` shows all of the above
 
-</details>
+# Releases
```

Comment:
I'd prefer to have the first level title at the top as it was with `# Changelog`, looks strange to have it here after a level 2 title.